event/csv_handler_test.go

```diff
@@ -90,7 +91,7 @@ func TestCsvHandler(t *testing.T) {
 		fileGetterStub := &eventtest.FileGetter{Reader: stubReadCloser}
 		observationWriterStub := &eventtest.ObservationWriter{}
 
-		eventHandler := event.NewCSVHandler(fileGetterStub, observationWriterStub)
+		eventHandler := event.NewCSVHandler(fileGetterStub, observationWriterStub, nil)
```

Comment:
Should there be some test cases added where we provide an error handler? Or is that all covered by the tests in the other package?

Reply:
I really struggled with this bit, I have tested it within the go-ns package I don't know if that's sufficient  enough tho